core/src/test/scala/integration/kafka/api/AdminClientIntegrationTest.scala

```diff
@@ -312,16 +312,16 @@ class AdminClientIntegrationTest extends IntegrationTestHarness with Logging {
   def testDescribeLogDirs(): Unit = {
     client = AdminClient.create(createConfig())
     val topic = "topic"
-    val leaderByPartition = createTopic(topic, numPartitions = 10, replicationFactor = 1)
```

Comment:
I think it's a good idea to keep this, even if it's a default value, since the test may rely on it. We don't want a lot of tests to break if somebody changes the default value in `createTopic()`